packages/snap-preact/README.md

```diff
@@ -252,9 +252,11 @@ const { search } = snap.controllers;
 ```
 
 ## properties
-
 After instantiating an instance of Snap, the following properties can be accessed. 
 
+### event-manager 
```

Comment:
I think the property name should match up with the subheading here - `eventManager`.